index.js

```diff
@@ -24,12 +26,26 @@ module.exports = async function hubdown (markdownString, opts = {}) {
   let data = {}
   let content = markdownString
 
+  // check the cache for preprocessed markdown
+  if (opts.cache) {
+    const existing = await opts.cache.get(hash).catch(err => {
```

Comment:
this is an interesting pattern, but also one hard to grasp. I would find this easier to read:

```js
let existing = false
try {
  existing = await opts.cache.get(hash)
} catch (err) {
  if (!err.notFound) console.error(err)
}
```

`err.notFound` is a way of checking if the error can be safely ignored, since it's just a "404".

Reply:
Thanks! Updated to make the code more intuitive.